### PR TITLE
https://issues.redhat.com/browse/ACM-3383--move topics, redo descript…

### PR DESCRIPTION
**Pull request by swopebe**
…ions where needed, move folders and remvoe items not needed

### CODE REVIEW

File: about/docinfo.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <productname>{product-title}</productname>
 <productnumber>{product-version}</productnumber>
-<subtitle>Read more about the product including the architecture, components, and quick start guide.</subtitle>
+<subtitle>Read more about the product including the architecture and components.</subtitle>
 <abstract>
-    <para>Read more about the product including the architecture, components, and quick start guide.</para>
+    <para>Learn about architecure and other specific information about {product-title}.</para>
```
Comment (dockerymick) [inline about/docinfo.xml:5]: In the later `.xml` file the subtitle and paragraph description are the same. Is there a reason why the paragraph isn't the same as the subtitle?

**Review by dockerymick — CHANGES_REQUESTED** [review]
Left a comment. Did you update link references? For example, changing a link reference from backup_restore/backup_activation.adoc to business_continuity/backup_restore/backup_activation.adoc. I did a search and received 33 results
Screenshot 2023-05-01 at 1 25 11 PM

File: about/docinfo.xml (hunk 1 above)
Comment (swopebe) [reply about/docinfo.xml:5]: fixed, subtitles changed.

**Review by dockerymick — APPROVED** [review]
thanks for the changes! this looks good to me !

**Pull request merged**